libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -55,10 +55,27 @@ const AP_Param::GroupInfo AP_Camera::var_info[] PROGMEM = {
     // @Values: 0:Low,1:High
     // @User: Standard
     AP_GROUPINFO("RELAY_ON",    5, AP_Camera, _relay_on, 1),
-    
+
+    // @Param: MIN_INTERVAL
+    // @DisplayName: Minimum time between photos
+    // @Description: Postpone shooting if previous picture was taken less than preset time ago.
+    // @User: Standard
+    // @Units: milliseconds
+    // @Range: 0 10000
+    AP_GROUPINFO("MIN_INTERVAL",  6, AP_Camera, _min_interval, 0),
+
+    // @Param: MAX_ROLL
+    // @DisplayName: Maximum photo roll angle.
+    // @Description: Postpone shooting if roll is greater than limit.
+    // @User: Standard
+    // @Units: Degrees
+    // @Range: 0 180
+    AP_GROUPINFO("MAX_ROLL",  7, AP_Camera, _max_roll, 45),
```

Comment:
The range in the above comment exceeds the max value storable in a signed 8 bit int. You need the storage to be unsigned 8 bit. And it would be nice to allow a value of 0 to disable the check, as setting 180 is non intuitve, and the default value changes the behavior of the system.